frontend/src/components/form/base/ENumberField.vue

```diff
@@ -14,11 +14,6 @@
   >
     <!-- passing through all slots -->
     <slot v-for="(_, name) in $slots" :slot="name" :name="name" />
-    <template #scoped="{ scopedSlots }">
```

Comment:
Why was this removed?

Reply:
Because it was not working correctly. I was not able to bind additional arguments to the slot, as it was the exact scoped slot of the `v-text-field`. I think it is sensible to not loop over the scoped slots anyway for vue3, as it works quite different there.